mqtt_exporter/main.py

```diff
@@ -326,14 +326,21 @@ def _parse_message(raw_topic, raw_payload):
     """Parse topic and payload to have exposable information."""
     # parse MQTT payload
     try:
-        payload = json.loads(raw_payload)
-    except json.JSONDecodeError:
-        LOG.debug('failed to parse payload as JSON: "%s"', raw_payload)
-        return None, None
+        if not isinstance(raw_payload, str):
```

Comment:
I don't understand when the payload could be seen as a non string object.

Reply:
> I don't understand when the payload could be seen as a non string object.

At least when I ran it in the debugger, it comes in as `bytes`, and needs to be decoded to a `str` (which happens on the next line).
Previously, the decoding was handled by `json.loads`, which accepts both `str` and `bytes`.